packages/webdriverio/src/utils/findStrategy.ts

```diff
@@ -270,16 +273,21 @@ export const findStrategy = function (selector: SelectorStrategy, isW3C?: boolea
             )
         }
         const partialNot = ` and not(${`.//${tag || '*'}${conditions.length ? `[${conditions.join(' and ')}]` : ''}`})`
-        conditions.push(
-            partial ? `contains(., "${query}")${partialNot}` : `normalize-space(text()) = "${query}"`
-        )
+        if (insensitive) {
+            conditions.push(
+                partial
+                    ? `contains(translate(., "ABCDEFGHIJKLMNOPQRSTUVWXYZ", "abcdefghijklmnopqrstuvwxyz"), "${query.toLowerCase()}")${partialNot}`
+                    : `normalize-space(translate(text(), "ABCDEFGHIJKLMNOPQRSTUVWXYZ", "abcdefghijklmnopqrstuvwxyz")) = "${query.toLowerCase()}"`)
+        } else {
+            conditions.push(partial ? `contains(., "${query}")${partialNot}` : `normalize-space(text()) = "${query}"`)
+        }
         const getValue = () => `.//${tag || '*'}[${conditions.join(' and ')}]`
         value = getValue()
         if (!partial) {
             conditions.pop()
             conditions.push(
                 `not(${value})`,
-                `normalize-space() = "${query}"`
+                `normalize-space() = "${insensitive? query.toLowerCase() : query}"`
```

Comment:
Could you explain why we lowercase this? Feels like this should not be needed as we already deal with it in the lines above

Reply:
Thanks for catching...

In my opinion, while it's case insensitive scenario the input seletor need to be sanitized to low key ex: 

`h1.=AnTHony`


should be translate to 

`.//h1[normalize-space(translate(text(), "ABCDEFGHIJKLMNOPQRSTUVWXYZ", "abcdefghijklmnopqrstuvwxyz")) = "anthony"] | .//h1[not(.//h1[normalize-space(translate(text(), "ABCDEFGHIJKLMNOPQRSTUVWXYZ", "abcdefghijklmnopqrstuvwxyz")) = "anthony"]) and normalize-space() = "anthony"]`

instead of 

`.//h1[normalize-space(translate(text(), "ABCDEFGHIJKLMNOPQRSTUVWXYZ", "abcdefghijklmnopqrstuvwxyz")) = "anthony"] | .//h1[not(.//h1[normalize-space(translate(text(), "ABCDEFGHIJKLMNOPQRSTUVWXYZ", "abcdefghijklmnopqrstuvwxyz")) = "anthony"]) and normalize-space() = "AnTHony"]`